app/src/main/res/layout/activity_navigation2.xml

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".Navigation">
+
+    <Button
+        android:id="@+id/workouts_button"
+        style="@style/Widget.AppCompat.Button.Borderless.Colored"
+        android:layout_width="0dp"
+        android:layout_height="67dp"
```

Comment:
Some of buttons have 67dp as height, others have "wrap_content". I think it should be the same for every button

Reply:
Fixed it, height for all the buttons is same now.

Reply:
Looks like the commit only included the change of the file name

Reply:
Now height should be fixed.